### PR TITLE
opt: more configurable scroll logic & edge size

### PR DESCRIPTION
**Pull request by Kingtous**

### CODE REVIEW

File: flutter/lib/desktop/widgets/scroll_wrapper.dart [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      {Key? key, required this.scrollController, required this.child})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ImprovedScrolling(
+      scrollController: scrollController,
+      enableCustomMouseWheelScrolling: false,
+      customMouseWheelScrollConfig:
+          const CustomMouseWheelScrollConfig(scrollAmountMultiplier: 3.0),
```
Comment (Kingtous) [inline flutter/lib/desktop/widgets/scroll_wrapper.dart]: 3.0 is default mouse wheel scroll multiplier. Can customed to fit different platforms.

Comment (rustdesk) [reply flutter/lib/desktop/widgets/scroll_wrapper.dart]: Use constant for all these variable.

Comment (Kingtous) [reply flutter/lib/desktop/widgets/scroll_wrapper.dart:17]: > Use constant for all these variable.

yes, it's already done.

Comment (rustdesk) [reply flutter/lib/desktop/widgets/scroll_wrapper.dart]: and 4.0

Comment (Kingtous) [reply flutter/lib/desktop/widgets/scroll_wrapper.dart]: done

Comment (Kingtous) [reply flutter/lib/desktop/widgets/scroll_wrapper.dart]: done

**Pull request merged**